app/src/main/java/org/oppia/android/app/view/ViewComponentImpl.kt

```diff
@@ -31,6 +32,7 @@ interface ViewComponentImpl : ViewComponent {
   fun inject(dragDropSortInteractionView: DragDropSortInteractionView)
   fun inject(imageRegionSelectionInteractionView: ImageRegionSelectionInteractionView)
   fun inject(lessonThumbnailImageView: LessonThumbnailImageView)
+  fun inject(continueButtonView: ContinueButtonView)
```

Comment:
These injections should be in alphabetical order--please update.

Reply:
done